gateways/http/account/list_account.go

```diff
@@ -6,15 +6,20 @@ import (
 	"net/http"
 )
 
+type Account struct {
+	ID        string
+	Name      string
+	CPF       string
+	Balance   float64
+	CreatedAt string
+}
```

Comment:
beware of of your json keys standard, if don't put any tags like this, the generated json will be like:
```json5
{
  "ID": "...",
  "Name": "...",
  "CPF": "...",
...
}
```

to avoid it, use golang json tags